docusaurus/docs/learn/core-concepts/security/enrollment.md

```diff
@@ -159,6 +159,140 @@ The default format is `[caName] - [commonName]`.
 
 Identity names are unique and if a collision occurs, incrementing numbers are appended.
 
+### Auto External JWT Signer Enrollment
+
+[External JWT Signers](./authentication/50-external-jwt-signers.md) represent an external IdP that can sign JWTs 
+as proof of identity (includinig OAuth/OIDC servers). Auto Ext-JWT-Signer enrollment allows a client with a JWT from a
+configured [External JWT Signers](./authentication/50-external-jwt-signers.md) to enroll without and administrator
+preconfiguring an identity for them.
+
+For this to function, the following must be true:
+- the External JWT Signer is configured to validate incoming JWTs
+- the External JWT Signer is enabled
+- the External JWT Signer has a `claimsProperty` set that maps to an id value to use as `externalId` of enrolling 
+    identities.
+- either the `enrollToCertEnable` or `enrollToTokenEnabled` are set to `true`
+
+
+The [External JWT Signers](./authentication/50-external-jwt-signers.md) options `enrollToCertEnable` and
+`enrollToTokenEnabled` are defined as follows:
+
+- `enrollToCertEnabled` - Allows clients with valid JWTs to request a certificate from the controller by providing a 
```

Comment:
this all seems like it's exactly the same content? if that's the case it would probably be easier to maintain as a docusaurus partial (_ext_jwt_details.md) or some such? (similar to what is done in `creating-service-policies.mdx`)